auth-web/tests/unit/components/AddNameRequestForm.spec.ts

```diff
@@ -1,17 +1,33 @@
+import { mount, shallowMount } from '@vue/test-utils'
 import AddNameRequestForm from '@/components/auth/manage-business/AddNameRequestForm.vue'
 import HelpDialog from '@/components/auth/common/HelpDialog.vue'
 import Vue from 'vue'
 import Vuetify from 'vuetify'
-import { shallowMount } from '@vue/test-utils'
+import flushPromises from 'flush-promises'
 
 Vue.use(Vuetify)
-
 const vuetify = new Vuetify({})
 
 // Prevent the warning "[Vuetify] Unable to locate target [data-app]"
 document.body.setAttribute('data-app', 'true')
 
 describe('Add Name Request Form', () => {
+  let wrapperFactory: any
+
+  beforeEach(() => {
+    wrapperFactory = (props: any) => {
```

Comment:
Why not set wrapper = wrapperFactory? 

instead of doing `const wrapper = wrapperFactory()` 6x

Reply:
updated